internals/secrethub/audit.go

```diff
@@ -63,13 +90,34 @@ func (cmd *AuditCommand) run() error {
 		return err
 	}
 
-	tabWriter := tabwriter.NewWriter(cmd.io.Stdout(), 0, 4, 4, ' ', 0)
-	header := strings.Join(auditTable.header(), "\t") + "\n"
-	fmt.Fprint(tabWriter, header)
+	var formatter tableFormatter
+	if cmd.json {
+		formatter = newJSONFormatter(auditTable.header())
+	} else {
+		terminalWidth, err := cmd.terminalWidth(int(os.Stdout.Fd()))
```

Comment:
Weirdly, I encountered strange behavior when using the `Stdout` from `ui.io`.

Reply:
Yes that's probably because `ui` gives you a TTY if it's in the terminal. But I think we should refrain from using `os.Stdout` and instead adapt the `ui` package to suit your needs. This you should definitely discuss/call about with @SimonBarendse

Reply:
Have you had a look at `io.Prompts` as an alternative to `io.Stdout`?

Is this still relevant? Give me a call if you'd like to discuss.

Reply:
After adapting the `ui` package passing `cmd.io.Stdout()` works as expected.